app/views/profiles/search.html.erb

```diff
@@ -0,0 +1,98 @@
+<div class = "search-page" id="friend-lookup">
+  
+  <div class = "col-md-6 col-md-offset-3">
+  	<div class="table-responsive">
+		  <table class="table search-table" >
+		    <thead>
+		    	<tr>
+		    		<th class="header search-header">Summoner Search Result</th>
+		    	</tr>
+		    </thead>
+		    <tbody>
+		      <tr>
+		    			<td>
+		    			  <% if @friend %>
+			    		    <div id="friend-lookup-results" class="row well results-block">
+			    		      <div class="col-sm-5">
+			    		      	<strong>Summoner Name: <%= @name.capitalize %></strong><br>
```

Comment:
As a rule of thumb use `<p>` instead of `<br>`.  `<br>` should only be used when you need a line break inside of a `<p>`.  You have a lot of these in the layout.